test-expected/sequelize/db-analysis-output/users.expected.json

```diff
@@ -30,7 +30,8 @@
         "foreignKey": "user_id",
         "foreignKeyName": "userIdKey",
         "junctionTable": "user_books",
```

Comment:
Remove `junctionTable` here if you delete the variable assignment on my other comment

Reply:
👍